docs/VictoriaLogs/CHANGELOG.md

```diff
@@ -35,6 +35,8 @@ Released at 2024-12-22
 * BUGFIX: [`sort` pipe](https://docs.victoriametrics.com/victorialogs/logsql/#sort-pipe): fix improper sorting of numeric fields in some cases.
 * BUGFIX: properly return an empty minimum value from [`min` stats function](https://docs.victoriametrics.com/victorialogs/logsql/#min-stats).
 
+* BUGFIX: [vlinsert](https://docs.victoriametrics.com/victorialogs/): order of VL-Msg-Field values now defines a priority of these fields and it's now obvious for a user which field will be picked if multiple msg_field values exist in a row.
```

Comment:
@AndrewChubatiuk , could you move this line to the `tip` chapter above?

Reply:
done